.github/workflows/php.yml

```diff
@@ -32,16 +76,19 @@ jobs:
         with:
           php-version: ${{ matrix.php }}
 
+      # Installing a specific version of Laravel will force us to compatible versions of `illuminate/http` and
+      # `illuminate/support`. If we didn't do this step then we'd always only ever test on the latest versions of those
+      # two packages.
       - name: Install dependencies
         run: |
           composer require "laravel/framework:${{ matrix.laravel }}" --no-interaction --no-update
-          composer update --${{ matrix.dependency-version }} --prefer-dist --no-interaction --no-suggest
+          composer update --prefer-stable --prefer-dist --no-interaction --no-suggest
```

Comment:
We're no longer testing against `--prefer-lowest` and `--prefer-stable` because `--prefer-lowest` was causing all sorts of problems with PHPUnit and Mockery on Laravel 9 on PHP 8.1 that it wasn't worth it to test both when testing against stable-only is good enough for our needs.